prov/rxd/src/rxd_cq.c

```diff
@@ -1064,6 +1065,7 @@ static void rxd_handle_start_data(struct rxd_ep *ep, struct rxd_peer *peer,
 repost:
 	rxd_ep_repost_buff(rx_buf);
 out:
+	assert(rxd_reposted_bufs);
```

Comment:
Thanks - I'm a fan of assertions, just wanted to ensure that this assertion wasn't outside of a race condition.